### PR TITLE
Servicebus: Geo DR feature

### PR DESCRIPTION
**Pull request by v-Ajnava**
## Description
Geo DR feature added
---
This change adds metadata Geo-Disaster recovery support to the Service Bus .NET management client. API are being added to create and manage a Disaster recovery configuration which allows users to pair two Service Bus namespaces in a primary-secondary configuration and creates an alias to represent the DR pair. Also, being added are APIs to trigger a Geo-DR failover which re-points the alias from the primary to the secondary namespace and APIs to break DR pairing which disables Geo-DR.

Swagger change PR Link : Azure/azure-rest-api-specs#1714

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [X] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [X] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [X] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [X] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [X] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Review by sphibbs — COMMENTED** [review]
We are there 3 deleted tests. looks like this PR is additive functionality.

**Comment by v-Ajnava** [comment]
@sphibbs, Those tests were to check the removal of maximum length constrain for Subscription and Authorizationrules. Removal of Max length constrain change was only for top level entities like Queues and Topics and reverted for Subscription and Authorizationrules. So deleted those tests.

File: src/SDKs/ServiceBus/ServiceBus.Tests/ServiceBus.Tests.csproj [hunk 1]
```diff
@@ -10,6 +10,12 @@
   <PropertyGroup>
     <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
   </PropertyGroup>
+
+  <ItemGroup>
```
Comment (shahabhijeet) [inline src/SDKs/ServiceBus/ServiceBus.Tests/ServiceBus.Tests.csproj]: Including ItemGroup for files is not necessary.
Please remove this ItemGroup as the new project system picks up all files

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Missing proper description for the PR.
Looking at the PR, there is no way to know what is being changed. Please be very clear as to what changes are going in. This is a public repo and contributors everywhere should know what is being checked in.
Please provide link to the swagger change PR (not the commit but the PR)
Also missing the metadata file.
Missing version bump for the SDK

**Comment by v-Ajnava** [comment]
@shahabhijeet, Swagger change PR Link : https://github.com/Azure/azure-rest-api-specs/pull/1714

File: src/SDKs/ServiceBus/ServiceBus.Tests/ServiceBus.Tests.csproj (hunk 1 above)
Comment (v-Ajnava) [reply src/SDKs/ServiceBus/ServiceBus.Tests/ServiceBus.Tests.csproj]: Removed.

**Comment by v-Ajnava** [comment]
@cormacpayne, have updated the PR description, version and metadata file, please let me know your comments.

**Comment by v-Ajnava** [comment]
@shahabhijeet, I am not able to change the label and have addressed all the comments.

File: src/SDKs/ServiceBus/Management.ServiceBus/Microsoft.Azure.Management.ServiceBus.csproj [hunk 2]
```diff
@@ -6,6 +6,6 @@
     <AssemblyName>Microsoft.Azure.Management.ServiceBus</AssemblyName>
     <PackageTags>Microsoft Azure ServiceBus Management;ServiceBus;ServiceBus management</PackageTags>      
     <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
-    <VersionPrefix>1.0.1</VersionPrefix>
+    <VersionPrefix>1.0.2</VersionPrefix>
```
Comment (shahabhijeet) [inline src/SDKs/ServiceBus/Management.ServiceBus/Microsoft.Azure.Management.ServiceBus.csproj]: Add < PackageReleaseNotes > and add Release notes.
I would recommend to use < ![CDATA and then format your Release notes

https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/MarketplaceOrdering/Management.MarketplaceOrdering/Microsoft.Azure.Management.MarketplaceOrdering.csproj#L10

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Please add release notes to EvenHub as well, similar to what is mentioned in this PR.

File: src/SDKs/ServiceBus/Management.ServiceBus/Microsoft.Azure.Management.ServiceBus.csproj (hunk 2 above)
Comment (v-Ajnava) [reply src/SDKs/ServiceBus/Management.ServiceBus/Microsoft.Azure.Management.ServiceBus.csproj]: updated package release notes

Comment (v-Ajnava) [reply src/SDKs/ServiceBus/Management.ServiceBus/Microsoft.Azure.Management.ServiceBus.csproj]: @shahabhijeet, have updated package release notes. can you please review it.

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**